source/leuven/wice_advanced_guide.rst

```diff
@@ -163,6 +163,55 @@ Also keep in mind that applying your ``~/.bashrc`` settings in your Slurm jobs
 requires placing ``#!/bin/bash -l`` at the top of your Slurm jobscript, as
 shown in the :ref:`wICE quickstart guide <running jobs on wice>`.
 
+.. _gpu_compute_mode:
+
+Setting the GPU compute mode
+----------------------------
+
+NVIDIA GPUs support multiple `compute modes
+<https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-modes>`_.
+By default, the compute mode is set to `Exclusive-process` on our clusters
+(which is the best setting in the majority of cases), but you can choose
+another compute mode at job submission time. This is done by making use of a
+plugin for our Slurm job scheduler:
+
+::
```

Comment:
I suggest to replace all the basic `::` with a code block to get syntax highlighting:
```suggestion
.. code-block:: shell
```

Reply:
Done in 0ad90459ed4001b9ae4c855d7af487dac0801a99